client/src/controllers/TagController.test.js

```diff
@@ -58,8 +58,6 @@ describe('TagController', () => {
     const [{ preprocessTag }] = tagitMock.mock.calls[0];
 
     expect(preprocessTag).toBeInstanceOf(Function);
-
-    expect(preprocessTag()).toEqual();
```

Comment:
We will need more tests added please, mocking the API request/respsonse via mocking the ajax result.

Reply:
I'll work on that, not really confident in testing with python but I'll do my best